### ISSUE TITLE
App::Cmd::Tester doesn't work when using IPC::Open3 in a command

### ISSUE THREAD

**Issue by preaction** [rjbs/App-Cmd#22]
Description / demonstration of the problem: https://gist.github.com/preaction/5550143

TL;DR: App::Cmd::Tester's use of IO::TieCombine gets untied if the command uses IPC::Open3.

Unfortunately, the only fixes possible on the App::Cmd::Tester side involve either removing the "output" key that shows the combined stdout/stderr, or making that the only output.

I tried using Capture::Tiny inside of App::Cmd::Tester, but it also changed what App::Cmd::Tester is capable of (removing either the combined or the uncombined output).

The only way forward I see right now is to create an Untied version of App::Cmd::Tester that allows the user a choice of split or combined output, but not both.

Issue also reported to System-Command: https://github.com/book/System-Command/issues/10
